pyproject.toml

```diff
@@ -30,10 +30,15 @@ dependencies = [
   "prometheus_client==0.20.0",
   "grpcio==1.64.1",
   "grpcio-health-checking==1.62.1",
-  "grpcio-reflection==1.64.1",
+  "grpcio-reflection==1.62.2",
```

Comment:
This is needed unfortunately because of version conflicts

Reply:
Does setting all grpc packages version to `1.64.1` work?

Reply:
Maybe it's worth it set all grpc packages to the same version, just to be on the safe side

Reply:
Let me try that

Reply:
Trying with 
```
(vllm) pip3 list | grep grpc
grpcio                                   1.62.2
grpcio-health-checking                   1.62.2
grpcio-reflection                        1.62.2
```

Reply:
Seems to work!